docs/js/layout.js

```diff
@@ -167,11 +167,14 @@ function fixupSidenotes() {
 
 function loadVersionOptions() {
 	var currentVersion
-	var matchedVersion = location.pathname.match('/docs/([0-9].*)/core')
+	var matchedVersion = location.pathname.match('/docs/([\d+\\.\d+)/')
```

Comment:
I think you need to double-slash the `\d`, as in `\\d`. The JS quoting rules will swallow the backslash otherwise, e.g.:

```
jeff@jeff-chain ~/go/src/chain (main) $ node
> console.log('\d')
d
```